### ISSUE TITLE
about  x_block_size and  y_block_size

### ISSUE THREAD

**Issue by l333wang** [Weixing-Zhang/NDVI-for-Large-Raster-Python#2]
I've noticed that you never use these  x_block_size and  y_block_size. Is that just for developer's reference?

**Comment by Weixing-Zhang** [comment]
@l333wang 
They are two of necessary variables. x_block_size and y_block_size indicate the number of block in terms of x and y. The reason I divided the whole image into blocks and then processed the RS image block by block was that my pc did not have RAM memory.

**Comment by l333wang** [comment]
I haven't seen you used these two variables after you defined them.  Am I lost something?

**Comment by l333wang** [comment]
I think these two variables are how many blocks on x and y, right?

**Comment by Weixing-Zhang** [comment]
@l333wang 
You are right. I am sorry about this. They are not necessary to run the code. They are the numbers of blocks on x and y. Let me delete them.

**Comment by l333wang** [comment]
But I agree that these two variables are good reference to know.
